### ISSUE TITLE
Error message: The saved file cannot be reopened.

### ISSUE THREAD

**Issue by kuenst** [kovzol/aris#22]

**Comment by kovzol** [comment]
Laura, please share more details. Do you mean that the saved file has no `.tle` extension and therefore it is not shown in the file chooser window? If so, maybe it should be forced to put the `.tle` extension before saving the file.

**Comment by kuenst** [comment]
Yes, there is no .tle extesion shown, so it would be good to add it automatically and if you add it manually, the file won´t open either.
